src/App.js

```diff
@@ -39,8 +39,12 @@ class App extends Component {
     console.log(this.state);
   }
 
-  onItemClicked = () => {
-    // Fill this in!
+  onItemClick = (item) => {
```

Comment:
Hm, I might want to call this parameter by a different variable name, like `itemType`, to distinguish it from the item itself